File: Fines2Model3/Values/Designacion.cs
namespace SqlOrganize.Sql.Fines2Model3
{
    public class DesignacionValues : EntityVal
    {
        public DesignacionValues(Db _db, string _entity_name, string? _field_id) : base(_db, _entity_name, _field_id)
        {
        }

        public override string ToString()
        {
            var s = "";

            EntityVal? v = GetValuesCache("persona");
            if(v != null)
            {
                s += v.GetStr("nombres", "?");
                s += " ";
                s += v.GetStr("apellidos", "?");
                s += " ";
                s += v.GetStr("telefono", "?");
            } else
            {
                s = "?";
            }

            return s.Trim();
        }
    }
}

    

File: src/Fines2/Model/Designacion_.php
<?php

namespace Fines2\Model;

use \Fines2\Model\Designacion;

use SqlOrganize\Sql\Entity;
use Exception;
use DateTime;

class Designacion_ extends Designacion
{
 public function getLabel(): string
    {
        return ($this->persona_?->nombres ?? "?"). " " . ($this->persona_?->apellidos ?? "?") . " " . ($this->persona_?->telefono ?? "?");
    }
}

